packages/cli/src/commands/app/serve.ts

```diff
@@ -14,13 +14,19 @@
  * limitations under the License.
  */
 
-import { serveBundle } from '../../lib/bundler';
+import fs from 'fs-extra';
 import { Command } from 'commander';
+import { serveBundle } from '../../lib/bundler';
+import { paths } from '../../lib/paths';
 
 export default async (cmd: Command) => {
+  const pkgPath = paths.resolveTarget('package.json');
+  const pkg = await fs.readJson(pkgPath);
```

Comment:
I think you can just put this inside serveBundle instead tbh, the options here are really for the differences between the serve commands and flags etc. The implementation is highly opinionated :grin:

Reply:
For the app I'd say we reeeally need different mechanism, this is very temp. For the plugin though it can be pretty ok solution during development, at least for simple CORS termination

Reply:
tbh I think that goes for plugin serve as well